packages/botonic-react/src/components/whatsapp-cta-url-button.tsx

```diff
@@ -0,0 +1,75 @@
+import { INPUT } from '@botonic/core'
+import React from 'react'
+
+import { truncateText } from '../util'
+import { renderComponent } from '../util/react'
+import { generateWebviewUrlWithParams } from '../util/webviews'
+import { Message } from './message'
+import {
+  WHATSAPP_MAX_BODY_CHARS,
+  WHATSAPP_MAX_BUTTON_CHARS,
+  WHATSAPP_MAX_FOOTER_CHARS,
+  WHATSAPP_MAX_HEADER_CHARS,
+} from './multichannel/multichannel-utils'
+import { whatsappMarkdown } from './multichannel/whatsapp/markdown'
+
+export const WHATSAPP_MAX_BUTTON_LIST_CHARS = 24
+export const WHATSAPP_MAX_BUTTON_LIST_DESCRIPTION_CHARS = 72
+export const WHATSAPP_MAX_BUTTON_LIST_ID_CHARS = 200
+
+export interface WhatsappCTAUrlButtonProps {
+  header?: string
+  body: string
+  footer?: string
+  displayText: string
+  url?: string
+  webview?: any
+  params?: any
+}
```

Comment:
better type with some similar to
```suggestion
export interface WhatsappCTAUrlButtonProps {
  header?: string
  body: string
  footer?: string
  displayText: string
  url: string
} | {
  header?: string
  body: string
  footer?: string
  webview: any
  params?: any
}
```